content/00.front-matter.md

```diff
@@ -5,7 +5,7 @@
 ##}
 
 _A published version of this manuscript from 04 April 2018 is available at <https://doi.org/10.1098/rsif.2017.0387>.
-An effort is underway to update the manuscript to capture the state of the field in 2019.
```

Comment:
@cgreene is it okay with you if I remove this part as well?

Reply:
Yes - thanks!